### PR TITLE
Dtp/mac readme updates

### PR DESCRIPTION
**Pull request by Dtphelan1**
Addresses #16.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
 This repo hosts [Synthea](https://github.com/synthetichealth/synthea)-generated
 sample [FHIR](https://www.hl7.org/fhir/) bulk export results,
-useful for testing downstream workflows.
+useful for testing downstream workflows. It also hosts a script for generating samples of
+custom sizes. See the [regenerating a dataset section below](#regenerating-a-dataset) for
+instructions on generating custom sized datasets.
 
 ## Downloads
 
+We've created several pre-zipped bundles of sample patients if you only
```
Comment (mikix) [inline README.md:11]: Sentence cuts off

File: generate.sh [hunk 2]
```diff
@@ -61,14 +61,20 @@ done
 # But we are interested in a bit more of a mix than that, so fake some emergency department visits.
 HIST_NOTE_TYPE='"system":"http://loinc.org","code":"34117-2","display":"History and physical note"'
 EMER_NOTE_TYPE='"system":"http://loinc.org","code":"34111-5","display":"Emergency department note"'
+# Ensures that OSX users are using gsed if they have it installed 
+SED=sed
+which gsed >/dev/null && SED=gsed 
 # This sed line will modify every 4th line
-sed -i "0~4s|$HIST_NOTE_TYPE|$EMER_NOTE_TYPE|" $OUTDIR/DocumentReference.ndjson
+SED -i "0~4s|$HIST_NOTE_TYPE|$EMER_NOTE_TYPE|" $OUTDIR/DocumentReference.ndjson
```
Comment (mikix) [inline generate.sh:68]: You'll need `$SED`

File: generate.sh [hunk 3]
```diff
@@ -61,14 +61,20 @@ done
...
 
 # Split each file to meet GitHub file limits (100MB per file is hard limit, but they complain at 50MB)
 echo "Splitting files into smaller ones..."
+# Ensures that OSX users are using gsplit if they have it installed 
+SPLIT=split
+which gsplit >/dev/null && SPLIT=gsplit
 for file in $OUTDIR/*; do
   resource=$(basename $file .ndjson)
-  split -d --additional-suffix .ndjson --suffix-length 3 --line-bytes 49m $file $OUTDIR/$resource.
+  SPLIT -d --additional-suffix .ndjson --suffix-length 3 --line-bytes 49m $file $OUTDIR/$resource.
```
Comment (mikix) [inline generate.sh:77]: `$SPLIT`

**Review by mikix — APPROVED** [review]

File: generate.sh (hunk 2 above)
Comment (Dtphelan1) [reply generate.sh:68]: tyty

File: README.md (hunk 1 above)
Comment (Dtphelan1) [reply README.md:11]: Just going to remove this line. It's vestigial from before I made edits to the introduction above.

**Pull request merged**